controllers/front/cron.php

```diff
@@ -46,7 +46,13 @@ public function postProcess()
                         '{lastname}' => '',
                         '{message}' => 'There are new security threats detected in your shop ' . Configuration::get('PS_SHOP_NAME') . '. You can check for more details at module ' . $this->module->displayName,
                     ];
-                    Mail::send((int) Context::getContext()->language->id, 'newsletter', 'New security threats detected', $template_vars, Configuration::get('PS_SHOP_EMAIL'));
+                    $emails = explode(',', Configuration::get('FOPPVS_NOTIFICATION_EMAILS'));
+                    if (empty($emails)) {
+                        $emails = [Configuration::get('PS_SHOP_EMAIL')];
+                    }
+                    foreach ($emails as $email) {
```

Comment:
foreach loop is not necessary for sending emails to multiple recipients. Better to use $emails directly.

Reply:
you're right, for some reason I remembered that 1.6 didn't have it, but it did :)